src/components/link/link.component.js

```diff
@@ -162,6 +164,10 @@ InternalLink.propTypes = {
    * @private
    */
   innerRef: PropTypes.object,
+  /** Aria label for accessibility purposes */
+  ariaLabel: PropTypes.string,
```

Comment:
Can you add these props into `link.d.ts`

Reply:
This still needs to be done